nix-tools-release.nix

```diff
@@ -60,11 +60,13 @@ let
   # add muslc/ghcjs/wasm, and other targets as needed.
   mapped-pkgs-mingw32 = mapTestOnCross lib.systems.examples.mingwW64 (nix-tools-pkgs [ builtins.currentSystem ]);
 
+  mapped-pkgs-ghcjs = mapTestOnCross lib.systems.examples.ghcjs (nix-tools-pkgs [ builtins.currentSystem ]);
+
   mapped-pkgs-all
     = lib.recursiveUpdate
         (mapped-pkgs)
-        (lib.mapAttrs (_: (lib.mapAttrs (_: (lib.mapAttrs' (n: v: lib.nameValuePair (lib.systems.examples.mingwW64.config + "-" + n) v)))))
```

Comment:
@hamishmack I think, we want to do some folding here. We'd most certainly want to retain the `mapped-pkgs-mingw32`.  Folding all cross packages onto the `mapped-pkgs` with `recursiveUpdate` seems like it could work.